src/fsharp/MSBuildReferenceResolver.fs

```diff
@@ -101,14 +34,14 @@ module internal MSBuildResolver =
     /// When targeting .NET 2.0-3.5 on Windows, we expand the {WindowsFramework} and {ReferenceAssemblies} paths manually
```

Comment:
FSharp.Core 4.0..0.0+ requires a matching mscorlib 4.0 reference.  The comment above is just out of date